pages/application.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faFacebook,
+    faInstagram,
+    faLinkedinIn,
+    faMediumM,
+    faTwitter,
+    faYoutube,
+  } from '@fortawesome/free-brands-svg-icons';
+import Head from 'next/head';
+
+import JoinUs from '../components/JoinUs';
+
+const ApplicationPageContainer = styled.div`
+  background: ${p => p.theme.colors.background};
+  height: fit-content;
+  position: relative;
+`;
+
+const BackgroundImage = styled.img`
+  position: absolute;
+  width: 510px;
+  right: 0;
+`;
+
+const FooterContainer = styled.div`
+    text-align: center;
+    padding: 64px 0;
+    width: 100%;
+    color: ${p => p.theme.colors.secondary};
+`;
```

Comment:
Nit: the indentation here should be consistent with the rest. If you pull on the latest main, you should get some prettier configs that would automatically fix this for you when you save your file. Feel free to hmu if you need any help configuring this.